### ISSUE TITLE
[bug]: PDF export currently broken in the lab

### ISSUE THREAD

**Issue by anna-puk** [freesewing/freesewing#2997]
### What seems to be the problem? 🤔

When using the lab, I cannot export any patterns. (I've tried several.) I get the same error in my own branch and develop (both on Windows) and in Gitpod as well. On Discord, other users report the same issue.

### Are you a FreeSewing patron? 😃

No, I am not 😞

### Additional context

To reproduce, start the lab, got to e.g. `titan`, go to `ExportThing` and click the 'SVG' button.

error message:

```
Unhandled Runtime Error

TypeError: plugin.plugin is undefined
Source

../../packages/core/src/pattern.mjs (494:8) @ getPluginName

  492 | } else {
  493 |   if (plugin.name) return plugin.name
494 |   if (plugin.plugin.name) return plugin.plugin.name
      |      ^
  495 | }
  496 | 
  497 | return false
```

**Comment by joostdecock** [comment]
Note that SVG export is fixed.
And while PDF export is still broken, you can export to SVG and then use our on-demand tiler: https://tiler.freesewing.org/

I'm going to assign this to @eriese as they wrote the PDF exporter in the frontend. No pressure :)

**Comment by eriese** [comment]
fixed by #3068
